README.md

```
@@ -1,3 +1,159 @@
+
+- 빨간색이 한나라, 초록색이 초나라이다.
+- 좌표계는 고정이다.
+    - 각 축의 번호는 0부터 시작한다.
+    - 시작 지점은 왼쪽 하단(0, 0)이다.
+- 게임은 초나라부터 시작된다.
+
+## 보드판
+
+- [x] 9 X 10 사이즈이다.
+- [x] 기물들의 좌표를 관리한다.
+    - [x] 어떤 기물인지는 모른다.
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 2; Actual: 4

(MD007, ul-indent)